### ISSUE TITLE
[Discussão] Adicionar página de contato

### ISSUE THREAD

**Issue by giovana-morais** [pyladies-brazil/br-pyladies-pelican#373]
Eu e @cecivieira estávamos conversando hoje sobre como as pessoas não sabem direito o que é o PyLadies Brasil e como entrar nos grupos pra tirar dúvidas etc, então pensamos sobre criar uma página com alguns mini tutoriais, tipo como entrar no Slack, como entrar na lista de email. 

Se ninguém se manifestar aqui por muito tempo, a gente vai entender isso como um "sim" e vamos botar a mão na massa, beijossssss

**Comment by pamellabiotec** [comment]
Concordo com vocês @giovana-morais e @cecivieira . E se quiser ajuda nos tutoriais, estou aqui disponível. Uma para me aperfeiçoar mais nessas ferramentas e tecnologias como também estar em ação dentro da comunidade.

**Comment by giovana-morais** [comment]
muito obrigada pela disponibilidade, @pamellabiotec! é mt bacana ver alguém querendo dar uma força pra gente. :heart: acho que uma primeira ajuda mt boa seria saber se cê sentiu dificuldade em entrar em contato com alguém do grupo ou mesmo de entrar nas nossas redes oficias, como o slack, o email e até mesmo o telegram, que não é a rede mais oficial. hahaha

**Comment by pamellabiotec** [comment]
No início eu sentir falta de uma página de contato quando acessei pela primeira vez o site da [Pyladies Brasil](http://brasil.pyladies.com/). Daí eu encontrei o canal aberto da Pyladies no telegram, depois dele, conseguir acessar ao grupo do telegram, dentro deste eu preenchi um formulário pinado que dava indicação do Slack da Pyladies. Cá estou eu falando contigo sobre esse trajeto.

**Comment by biancarosa** [comment]
Acho que faz super sentido! Isso conversa um pouco com o que eu acho que o grupo de docs do pyladies deveria tocar, o que tu acha da gente fazer o quickstart (tipo, prover a página, a documentaçao de como contribuir) e tentar levar esses pontos pra galera que for tocar essas documentações (que no final das contas pode ser a gente mesmo) pra alimentar essa página?

**Comment by giovana-morais** [comment]
acho daora. vou tentar encaminhar isso logo depois que terminarmos o mapa. talvez até lá o grupo responsável pela documentação já esteja melhor definido e a gente consiga dividir o trabalho (:

**Comment by cecivieira** [comment]
A ideia do quickstart é massa!
Acho que eu teria desistido de entrar em contato se tivesse que fazer essa jornada de Pâmella.

**Comment by biancarosa** [comment]
Pensem em adicionar uma pagina de contato com o email do pyladies brasil, a link das nossas docs, etc
